### ISSUE TITLE
New Import method issue

### ISSUE THREAD

**Issue by b0bbybacc** [jayjzheng/ff_rankings#15]
I am not even able to download your sample .csv and then import it back into the system without the following error:

Error: Unsupported File Type: application/vnd.ms-excel 

It was not converted. touched in anyway it is still a csv. Tried with the latest Chrome and Firefox browsers, both yield the same error.

**Comment by supershick** [comment]
yup, getting the same thing. it doesn't matter what file extension it is, not getting accepted.

**Comment by jayjzheng** [comment]
it might be due to that a system with Excel changing csv's file type. I can't verify that since I don't have a excel. got a temporary fix in. will redo it in the future.
